packages/shared/src/config/global.ts

```diff
@@ -1,3 +1,4 @@
 export const global = {
   appName: 'Hangar',
+  authReturnUriOverride: 'returnTo',
```

Comment:
I'd prefer something like `authReturnToQueryKey`... thoughts?

Reply:
i mean, i would not understand that... `returnTo` is ambiguous, and i wouldn't even know what to make of `QueryKey`; whereas `returnUri` calls out the purpose, and `override` signals that it is optional.